### ISSUE TITLE
For `tab bar` component, the selected item is not contrasted enough for `Orange` theme

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1278]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone

### Device OS version

26

### OUDS iOS library version

1.1.0

### Describe the issue

<img width="150" height="74" alt="" src="https://github.com/user-attachments/assets/60fd2df7-1360-4f14-9f1e-1216ae6b03dc" />

The selected item is orange #E14E15 on grey background #DFDFDF
Text contrast is under 4.5:1 
Contrast ratio of 3:1 is acceptable for non-text elements only.

### Expected Behavior

Ensure a sufficient contrast for texts, by using black and bold color for example instead of orange
Orange can be still used for picto

### Accessibility settings

None
